app/components/contact-modal/contact-modal.tsx

```diff
@@ -7,11 +7,10 @@ import { useI18nContext } from "@app/i18n/i18n-react"
 import { openWhatsApp } from "@app/utils/external"
 import { Icon, ListItem, makeStyles, useTheme } from "@rneui/themed"
 
-import TelegramOutline from "./telegram.svg"
 
 export const SupportChannels = {
   Email: "email",
-  Telegram: "telegram",
```

Comment:
it is better practice to comment out this line instead of deleting it completely, unless we are absolutely sure we we not going to use that code again.